xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -2666,6 +2669,7 @@ void CActiveAE::LoadSettings()
   m_settings.atempoThreshold = settings->GetInt(CSettings::SETTING_AUDIOOUTPUT_ATEMPOTHRESHOLD) / 100.0;
   m_settings.streamNoise = settings->GetBool(CSettings::SETTING_AUDIOOUTPUT_STREAMNOISE);
   m_settings.silenceTimeoutMinutes = settings->GetInt(CSettings::SETTING_AUDIOOUTPUT_STREAMSILENCE);
+  m_settings.mixSubLevel = settings->GetInt(CSettings::SETTING_AUDIOOUTPUT_MIXSUBLEVEL) / 100.0;
```

Comment:
I don't clamp here for a reason ... uses with the 200% bass need ... can just edit their scan and shoot in their foot :-)

Reply:
Sorry if this is obvious, by 
> can just edit their scan

do you mean manually adjusting the settings.xml?

In the past week I have finally watched a few movies with `mixsublevel` and depending on genre a setting of `150-200` worked very well (and is usually less "overkill" than the number would indicate, when the LFE is often stored at -10dB). Thanks again for the patch! I've already had a lot of fun with the preserved bass-effects in this build.

Reply:
Yeah - that's what I meant. I don't want to make it harder for the people with that need. While I want to stick to the default sqrt(2) values - especially when you make two speakers from one sub.